utils/utils.jsx

```diff
@@ -1761,3 +1761,7 @@ export function getSortedUsers(reactions, currentUserId, profiles, teammateNameD
 
     return {currentUserReacted, users};
 }
+
+export function getAnnouncementBarCount(state) {
```

Comment:
This can go in something like `selectors/views/announcement_bar` or even just in `default_announcement_bar/index` to keep it with the only code that uses it. I'm still trying to avoid adding more code to `utils.jsx` since it has so much random stuff in it.